recipes/build.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
```

Comment:
You can move this one liner into meta.yml

Reply:
Could you please let me know when I could I put it in the meta.yaml file?